### PR TITLE
Test ImuTracker 

### PR DESCRIPTION
**Pull request by gaschler**
Adds unit tests for ImuTracker to cover filtering of the gravity
vector and tracking angular velocity.

### CODE REVIEW

File: cartographer/mapping/imu_tracker_test.cc [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+#include "cartographer/mapping/imu_tracker.h"
+
+#include "cartographer/common/make_unique.h"
+#include "gtest/gtest.h"
+
+namespace cartographer {
+namespace mapping {
+namespace {
+
+const double kDuration = 3.f;
```
Comment (cschuet) [inline cartographer/mapping/imu_tracker_test.cc]: make `constexpr`?

File: cartographer/mapping/imu_tracker_test.cc [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+const double kPrecision = 1e-8;
+const int kSteps = 10;
+
+class ImuTrackerTest : public ::testing::Test {
+ protected:
+  void SetUp() override {
+    imu_tracker_ = common::make_unique<ImuTracker>(0.1 * kDuration, time_);
```
Comment (cschuet) [inline cartographer/mapping/imu_tracker_test.cc]: Please document meaning of this constant, either with a comment on  a descriptive constant name.

**Comment by gaschler** [comment]
Done with the corrections, PTAL.

**Comment by gaschler** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user gaschler. Merge queue now has a length of 1.

**Pull request merged**